ariadne_codegen/client_generators/result_types.py

```diff
@@ -405,6 +406,11 @@ def _process_field_implementation(
         if is_union(field_implementation.annotation):
             keywords[DISCRIMINATOR_KEYWORD] = generate_constant(TYPENAME_ALIAS)
 
+        if keywords and isinstance(field_implementation.value, ast.Constant):
```

Comment:
One thing to ask here, why are you adding this default only if the keywords is not empty? There should be some dependency on default when adding it to the keyword? If yes, can you leave a comment in the code?

Reply:
Because we add the `Field` call only when keyword is not empty.  If there is no `Field` then there is no need for `default` keyword arg that goes into `Field`.

ref: https://github.com/mirumee/ariadne-codegen/pull/292/files#diff-9042b649db08641441c03575181ff29e2d7b821c5de5cf07254fc4845bca9509R414

Reply:
`Field` is only added when property has alias.